app/controllers/api/v1/trips_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::TripsController < ApplicationController
+
+  def create
+    destination = get_destination(params)
+    trip = Trip.create(destination: destination,
+      lat: params[:lat],
+      lng: params[:lng])
```

Comment:
I don't get it. This is a major change from the previous review round. So are you NOT passing in the destination from the client side?

Reply:
Yes major rewrite, second one. had Cors issues with that server for the client side.